manifests/params.pp

```diff
@@ -17,7 +17,7 @@
       $rubygems_package = 'rubygems'
       $ruby_dev         = 'ruby-devel'
       $rubygems_update  = false
-      if $::operatingsystemmajrelease == 5 {
+      if $::operatingsystemmajrelease and versioncmp($::operatingsystemmajrelease, '5') == 0 {
```

Comment:
Is this necessary? is there a circumstance where `$::operatingsystemmajrelease` would ever be undef/nil?